toolkit/tools/imagecustomizerapi/os_subrelease.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type OSSubrelease struct {
```

Comment:
I'm thinking we don't have to map the names 1:1. Honestly some of those are very confusing like BuilderName or IDLike. It would be better to substitute with ones that are easier to understand like "Team" instead of "BuilderName"